main.go

```diff
@@ -511,7 +511,8 @@ func main() {
 
 	qanClient := getQANClient(ctx, db, *qanAPIAddrF)
 
-	agentsRegistry := agents.NewRegistry(db, prometheus, qanClient)
+	isDebugMode := *debugF || *traceF
```

Comment:
In other places we use this:
```go
l.Logger.GetLevel() >= logrus.DebugLevel
```

This makes that check request-scoped if there is a request with a scope.

Reply:
I haven't seen that. I did it this way because I found this https://github.com/percona/pmm-managed/pull/301/files#diff-7ddfb3e035b42cd70649cc33393fe32cR561. Using your way might be better.